flocker/provision/_aws.py

```diff
@@ -297,66 +297,6 @@ def get_ssh_key(self):
         # https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_KeyPairInfo.html
         return None
 
-    def create_node(self, name, distribution, metadata={}):
-        size = self._default_size
-        disk_size = 8
-
-        with start_action(
-            action_type=u"flocker:provision:aws:create_node",
-            name=name,
-            distribution=distribution,
-            image_size=size,
-            disk_size=disk_size,
-            metadata=metadata,
-        ):
-
-            metadata = metadata.copy()
-            metadata['Name'] = name
-
-            disk1 = EBSBlockDeviceType()
-            disk1.size = disk_size
-            disk1.delete_on_termination = True
-            diskmap = BlockDeviceMapping()
-            diskmap['/dev/sda1'] = disk1
-
-            images = self._connection.get_all_images(
-                filters={'name': IMAGE_NAMES[distribution]},
-            )
-            # Retry several times, no sleep between retries is needed.
-            instance = poll_until(
-                lambda: self._get_node(images[0].id, size, diskmap, metadata),
-                repeat(0, 10),
-                lambda x: None)
-            return AWSNode(
-                name=name,
-                _provisioner=self,
-                _instance=instance,
-                distribution=distribution,
-            )
-
-    def _get_node(self, image_id, size, diskmap, metadata):
-        """
-        Create an AWS instance with the given parameters.
-
-        Return either boto.ec2.instance object or None if the instance
-        could not be created.
-        """
-
-        with start_action(
-            action_type=u"flocker:provision:aws:get_node",
-        ) as context:
-            [instance] = self._run_nodes(1, image_id, size, diskmap)
-            context.add_success_fields(instance_id=instance.id)
-
-            poll_until(lambda: self._set_metadata(instance, metadata),
-                       repeat(1, INSTANCE_TIMEOUT))
-            try:
-                _wait_until_running(instance)
-                return instance
-            except FailedToRun:
-                instance.terminate()
-                return None     # the instance is in the wrong state
-
     def _run_nodes(self, count, image_id, size, diskmap):
```

Comment:
An example of the naming issues I mentioned in another comment. This function is creating instances and the name doesn't quite reflect that.